deploy/lvm-operator.yaml

```diff
@@ -246,6 +246,118 @@ status:
   conditions: []
   storedVersions: []
 
+
+##############################################
+###########                       ############
+###########     LVMNode CRD       ############
+###########                       ############
+##############################################
+
+# LVMNode CRD is autogenerated via `make manifests` command.
+# Do the modification in the code and run the `make manifests` command
+# to generate the CRD definition
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.8
```

Comment:
We can start using the controller-gen verison v0.4.0 version which will basically generates the `v1` CRD version by default.